<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DistanceAndLength</class>
 <widget class="QDialog" name="DistanceAndLength">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Distance and Length</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_D_right">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>25</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_D_Left">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>25</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxDistance">
   <property name="geometry">
    <rect>
     <x>121</x>
     <y>25</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_Left">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>100</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Millimeters</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_Right">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>5</y>
     <width>100</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Centimeters</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_clearLenght">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>82</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
